language/tools/move-cli/tests/testsuite/module_run_view/args.exp

```diff
@@ -6,10 +6,10 @@ Changed resource(s) under 2 address(es):
   Changed 1 resource(s) under address 0000000000000000000000000000000b:
     Added type 00000000::Test::R: [U64(7)]
 Discarding changes; re-run with --commit if you would like to keep them.
-Command `view move_data/0x0000000000000000000000000000000a/0x00000000000000000000000000000002::Test::R`:
-Resource not found.
-Command `view move_data/0x0000000000000000000000000000000b/0x00000000000000000000000000000002::Test::R`:
-Resource not found.
+Command `view move_data/0x0000000000000000000000000000000a/resources/0x00000000000000000000000000000002::Test::R`:
+Not found.
```

Comment:
Possible regression? Shouldn't this say "Resource not found."?

Reply:
Ah, good call. Not sure if that check is needed; let me try eliminating it.